### PR TITLE
cmd,client,daemon,caps,docs,po: remove capabilities

### PR DESCRIPTION
**Pull request by zyga**
Capabilities have been entirely replaced by skills. Let's remove them.

### CODE REVIEW

**Comment by chipaca** [comment]
adieu! :+1:

File: daemon/api_test.go [hunk 1]
```diff
@@ -1246,213 +1245,6 @@ func (s *apiSuite) TestInstallLicensedIntegration(c *check.C) {
...
-	c.Check(rsp.Type, check.Equals, ResponseTypeError)
-	c.Check(rsp.Status, check.Equals, http.StatusNotFound)
-	// Verify (internal)
-	after := d.capRepo.All()
-	c.Check(before, check.DeepEquals, after)
-}
-
-// Tests for GET /2.0/skills
-
-func (s *apiSuite) TestGetSkills(c *check.C) {
```
Comment (stevenwilkin) [inline daemon/api_test.go]: This test shouldn't be deleted?

Comment (zyga) [inline daemon/api_test.go]: Whoops, overager rm -rf :). Thanks.

**Comment by stevenwilkin** [comment]
Loving those deletions :+1:

**Pull request merged**